runtime/src/identity.rs

```diff
@@ -89,6 +89,48 @@ pub struct ClaimValue {
     pub value: Vec<u8>,
 }
 
+#[derive(codec::Encode, codec::Decode, Default, Clone, PartialEq, Eq, Debug)]
+/// A structure for passing claims to `add_claims_batch`. The type argument is required to be
+/// `timestamp::Trait::Moment`.
+pub struct ClaimRecord<U> {
+    did: IdentityId,
```

Comment:
OK, I can revert to making the members public. I used a constructor and view methods because it's regarded as better practice. But it doesn't indeed look well well with other structs that publish their fields.